src/elements/wrappers/ContentPreview.js

```diff
@@ -10,6 +10,14 @@ import ES6Wrapper from './ES6Wrapper';
 import ContentPreviewResponsive from '../content-preview';
 
 class ContentPreview extends ES6Wrapper {
+    /**
+     * Helper to programmatically refresh the preview's sidebar panel
+     * @returns {void}
+     */
+    refreshSidebar(): void {
```

Comment:
Do we also need to expose `refresh` in the ContentSidebar ES6 wrapper?

Reply:
That one already exists: https://github.com/box/box-ui-elements/blob/master/src/elements/wrappers/ContentSidebar.js#L17